econml/iv/dr/_dr.py

```diff
@@ -629,19 +2061,23 @@ def prel_model_effect(self, value):
 
 class LinearIntentToTreatDRIV(StatsModelsCateEstimatorMixin, IntentToTreatDRIV):
     """
-    Implements the DRIV algorithm for the intent-to-treat A/B test setting
+    Implements the DRIV algorithm for the Linear Intent-to-Treat A/B test setting
 
     Parameters
     ----------
-    model_Y_X : estimator
+    model_y_xw : estimator
         model to estimate :math:`\\E[Y | X]`.  Must support `fit` and `predict` methods.
 
-    model_T_XZ : estimator
+    model_t_xwz : estimator
         model to estimate :math:`\\E[T | X, Z]`.  Must support `fit` and `predict_proba` methods.
 
     flexible_model_effect : estimator
         a flexible model for a preliminary version of the CATE, must accept sample_weight at fit time.
 
+    z_propensity: float or "auto", optional, default "auto"
```

Comment:
We should allow for a model too. We should also mention that if 'auto' we assume that the instrument is fully randomized and independent of any other variables (e.g. an A/B test). So the proportion of Z=1 in the overall population is used as propensity.